Resources/Prototypes/Loadouts/backpacks.yml

```
@@ -280,3 +280,17 @@
+  id: LoadoutBackpackSatchelCE
+  category: Backpacks
+  cost: 0
+  exclusive: true
+  items:
+    - ClothingBackpackSatchelCE
+  requirements:
+    - !type:CharacterItemGroupRequirement
+      group: LoadoutBackpacks
+    - !type:CharacterJobRequirement
+      jobs:
+        - ChiefEngineer
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)